schemas/com.mailgun/message_clicked/jsonschema/1-0-0

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Mailgun message clicked event",
+  "self": {
+    "vendor": "com.mailgun",
+    "name": "message_clicked",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "campaign-id": {
+      "description": "id of campaign triggering the event.",
+      "type": "string"
+    },
+    "campaign-name": {
+      "description": "The name of campaign triggering the event.",
+      "type": "string"
+    },
+    "city": {
+      "description": "Name of the city the event came from or ‘unknown’ if it couldn’t be determined.",
+      "type": "string",
+      "maxLength": 64
+    },
+    "client-name": {
+      "description": "Name of the client software, e.g. ‘Thunderbird’, ‘Chrome’, ‘Firefox’.",
+      "type": "string",
+      "maxLength": 64
+    },
+    "client-os": {
+      "description": "OS family running the client software, e.g. ‘Linux’, ‘Windows’, ‘OSX’.",
+      "type": "string",
+      "maxLength": 64
+    },
+    "client-type": {
+      "description": "Type of software the link was opened in, e.g. ‘browser’, ‘mobile browser’, ‘email client’.",
+      "type": "string",
+      "maxLength": 64
+    },
+    "country": {
+      "description": "Two-letter country code (as specified by ISO3166) the event came from or ‘unknown’ if it couldn’t be determined.",
+      "type": "string",
+      "maxLength": 2
+    },
+    "device-type": {
+      "description": "Device type the link was clicked on. Can be ‘desktop’, ‘mobile’, ‘tablet’, ‘other’ or ‘unknown’.",
+      "type": "string",
+      "maxLength": 64
+    },
+    "domain": {
+      "description": "Domain that sent the original message.",
+      "type": "string",
+      "maxLength": 256
+    },
+    "event": {
+      "description": "Event name ('clicked').",
+      "type": "string",
+      "maxLength": 16
+    },
+    "ip": {
+      "description": "IP address the event originated from (currently only seems to support ipv4).",
+      "type": "string",
+      "format": "ipv4"
+    },
+    "mailing-list": {
+      "description": "The address of mailing list the original message was sent to.",
+      "type": "string"
+    },
+    "message-id": {
+      "description": "The Id of the message that bounced.",
+      "type": "string",
+      "maxLength": 1024
+    },
+    "recipient": {
+      "description": "Recipient who clicked.",
```

Comment:
Is it id or email?

Reply:
@chuwy I have changed all those to email